en_US/extensions/plugins.md

```diff
@@ -162,6 +164,154 @@ make rel
 
 This created a new EMQX plugin tarball  `my_emqx_plugin-1.0.0.tar.gz` that you can now upload and install to your running EMQX cluster.
 
+#### Write Config Schema for the plugin (Optional)
+
+{% emqxce %}
+:::tip
+EMQX Enterprise Edition features. EMQX Enterprise Edition provides comprehensive coverage of key business scenarios, rich data integration, product-level reliability, and 24/7 global technical support. Experience the benefits of this [enterprise-ready MQTT messaging platform](https://www.emqx.com/en/try?product=enterprise) today.
+:::
+{% endemqxce %}
+
+{% emqxce %}
+::: tip Tip
+EMQX Enterprise Edition features. EMQX Enterprise Edition provides comprehensive coverage of key business scenarios, rich data integration, product-level reliability, and 24/7 global technical support. Experience the benefits of this [enterprise-ready MQTT messaging platform](https://www.emqx.com/en/try?product=enterprise) today.
+:::
+{% endemqxce %}
+
+We introduced Avro Schema for plugins in EMQX Enterprise Edition 5.7.0 to provide the ability to update plugin configurations at runtime.
+Furthermore, you can declare UI in this schema, allowing the EMQX Dashboard to render configuration forms for ease of use.
```

Comment:
The initial load of the plugin config seems to be missed in the feature implementation ?

```suggestion
Starting with version 5.7, the EMQX plugin framework pre-loads JSON or HOCON format configuration file for each plugin before startup and validates them against an optional schema, if provided. The configuration schema should be described using an Avro schema, located in the file priv/plugin_name.avsc. Additionally, configurations can be updated at runtime by utilizing the management API with the request PUT /plugins/:name/config.

For the EMQX Enterprise edition, users have the option to include $ui metadata within the Avro schema objects. This metadata aids in generating a configuration management page on the dashboard, enhancing user interaction and ease of management.
```